### ISSUE TITLE
Bilerot Burrow: The Bile Lord boss porting issue

### ISSUE THREAD

**Issue by bloodmasked** [WarEmu/WarBugs#18677]
The last boss in BB is meant to pull everyone into his stomach, he did not pull our healers in.

**Comment by lilimx** [comment]
Hey,

thank you for your ticket. 
Do you have some screenshots?
What happened before, right in the stomach and after the pull?
Any errors? Weird npc behaviour....maybe even right after you enter the instance.

**Comment by bloodmasked** [comment]
The last few runs I did had a similar problem a couple times, when you get to the stomach stage... everyone is pulled in as they should be. However, once you kill his innards you are supposed to be teleport back out to finish the boss. But nobody gets ported out so we have to send 5 people to book out to IC and return to the dungeon. Then the remaining person books out once the 5 others arrive, once all 6 are back we finish the boss. (I am not entirely sure if you have to go through the steps of sending 5 people and then 1, but really didn't want to waste peoples time if we had to redo the boss from testing theories).

**Comment by Zomega-WAR** [comment]
Update from today's run from hell.

First attempt at Bilelord didn't trigger the "suck" effect but we were teleported inside. We killed the insides and were not teleported out. One person remained inside while the other 5 used their book/scroll to leave and re-enter the dungeon as supposedly keeping one person in the stomach ensures the boss is not reset. However, upon arrival the boss was at 100% again with someone in its stomach.

That person then teleported out and came back to the dungeon, and we attempted to kill it again. This time the boss was frozen in place and could not be moved, meaning he was just spawning puddles all around himself that we couldn't work around. We eventually got him down to low health (from 100% and with unavoidable puddles) and instead of being sucked in, the party was punted away instead (I'm guessing it triggered the punt that should've happened after being ejected from his stomach which hadn't happened). After the punt we were teleported into its stomach. We then killed the insides again, and were stuck inside again.

Once again, we left one person in while everyone else use scrolls to get out. The 5 people eventually got back to the boss and this time his health remained at 20% ish. The final person used a scroll to get out and then rejoined the party, and at that point the boss was finally killed.
